File: pgm/ABC_Java/ABC_191_A.java
import java.io.*;
// import java.math.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final String[] arr = reader.readLine().split (" ");
        final String Yes = "Yes";
        final String No = "No";
        final int V = Integer.parseInt (arr[0]);
        final int T = Integer.parseInt (arr[1]);
        final int S = Integer.parseInt (arr[2]);
        final int D = Integer.parseInt (arr[3]);
        System.out.println (V * T <= D && D <= V * S ? No : Yes);
        return;
    }
}

File: pgm/ABC_Kotlin/ABC_191_A.kt
// import java.io.BufferedOutputStream
// import java.io.BufferedReader
// import java.io.DataInputStream
// import java.io.FileInputStream
// import java.io.IOException
// import java.io.InputStreamReader
// import java.io.PrintWriter
// import java.lang.StringBuilder
// import java.math.BigDecimal
// import java.math.RoundingMode
// import java.time.LocalTime
// import java.time.format.DateTimeFormatter
// import java.util.Scanner
// import java.util.StringTokenizer
// import java.util.TreeMap

// import kotlin.math.*

const val Yes: String = "Yes"
const val No: String = "No"
const val space: String = " "

fun main() {
    val (V: Int, T: Int, S: Int, D: Int) = readln().split(space).map { it.toInt() }
    if (V * T <= D && D <= V * S) {
        println(No)
    } else {
        println(Yes)
    }
    kotlin.system.exitProcess(0)
}